src/vcpkg/portfileprovider.cpp

```diff
@@ -319,13 +309,12 @@ namespace vcpkg
                     auto ports_spec = ports_dir / port_name;
                     if (Paragraphs::is_port_directory(m_fs, ports_spec))
                     {
-                        auto found_scf = Paragraphs::try_load_port_required(m_fs, port_name, ports_spec);
-                        if (auto scfp = found_scf.get())
+                        auto found_scfl = Paragraphs::try_load_port_required(m_fs, port_name, PortLocation{ports_spec});
```

Comment:
Is there some duplicate work between `is_port_directory` and `try_load_port_required` that can be removed without much contortion?

Reply:
Yes, but not in this extraction :)